activities/workshops/index.md

```diff
@@ -6,5 +6,7 @@ type: Index
 
 Here is a list of workshops developed or used by the Foundation for Public Code:
 
-* [Governance exercise](governance-exercise.md)
-* [Governance game](governance-game/index.md)
```

Comment:
These changes effectively gut this section. Would you expect new workshops to also be in a file elsewhere but linked from here (for example, where would [a product marketing workshop](https://github.com/publiccodenet/about/blob/marketing-website-MVP-workshop/activities/workshops/codebase-marketing-website-workshop.md) live?
More broadly, do you propose that all templates be located in the folder with the activity that requires them, but potentially also linked from any other location the template could be relevant?

Reply:
Is this resolved with my comment and commits since this post?

Reply:
Yes, this clear now.